packages/carbon-graphs/src/js/core/BaseConfig/helper.js

```diff
@@ -1,10 +1,16 @@
 'use strict';
 
 import * as d3 from 'd3';
+import uuidv4 from 'uuid/v4';
 import { AXIS_TYPE, LINE_TYPE } from '../../helpers/constants';
 import errors from '../../helpers/errors';
 import utils from '../../helpers/utils';
 
+/**
+ * Creates random ID
+ */
+const idRef = () => uuidv4();
```

Comment:
imo there's no benefit with abstracting the `uuidv4` function within another function, because we are not doing any extra logic. i think it's much clearer to directly use the `uuidv4` function instead - thoughts?

Reply:
Fixed [here](https://github.com/cerner/terra-graphs/pull/117/commits/fccb8fb24d10df112091db185401cc0916207485#diff-606396e5aa13b62e41638acfeb1622b804a2c42a86dcf491ae3c52c60fd49389R61).